scripts/checkChecksums.ts

```diff
@@ -0,0 +1,63 @@
+import { execSync } from "child_process";
+import { dirname, join } from "path";
+import { exit } from "process";
+
+const a: string[] = execSync(
+  "grep -Rw docs/src/ -A 1 -e 'load:' | sed -E 's/--//'"
+).toString().split('\n').filter(line => line !== '');
+
+let errorString: string = '';
+
+let index: number = 0;
+while (index < a.length) {
+  const currentLine: string = a[index];
+  const docsFile: string = currentLine.split(':')[0];
+  const r1: RegExpMatchArray | null = currentLine.match(
+    /load: (.+)/
+  );
+  if (r1) {
+    const [ _fullMatch, programPath ] = r1;
+    const nextLine: string = a[index + 1];
+    const r2: RegExpMatchArray | null = nextLine.match(
+      /md5: (.+)/
+    );
+    if (r2) {
+      const topDir: string = dirname(dirname(__filename));
+      const actualMD5WithNewline: string = execSync(`md5sum ${
+        join(topDir, programPath)
+      } | awk '{print $1}'`).toString();
+      const actualMD5: string = actualMD5WithNewline.slice(0, -1);
+      const [ _fullMatch, md5InDocs ] = r2;
+      if (actualMD5 !== md5InDocs) {
+        errorString += programPath + '\n';
```

Comment:
I wouldn't bother with errors like this. Throw JSON blobs like the other errors do

Reply:
I'm not exactly sure what you mean by
> Throw JSON blobs like the other errors do

[5b0bfdf](https://github.com/reach-sh/reach-lang/pull/1028/commits/5b0bfdfecff32a370199177eb6969ee4d8460754) just `throw`s an `Error` if an `md5` is missing or doesn't match on `make`.

How far is that from what we want?